src/blocks/cards.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  // height: 406px;
+
+  width: 200px;
+  background-color: $four-color;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__top {
+    padding-left: 3px;
+    padding-right: 3px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    background-repeat: no-repeat;
+    background-size: auto;
```

Comment:
The `background-size` property is set to `auto`, which means the image will not be resized. If you intend to resize the image to fit its container, consider using `cover` or `contain` instead.